moda-contracts/src/interfaces/ICatalog.sol

```diff
@@ -0,0 +1,336 @@
+// SPDX-License-Identifier: MIT
+pragma solidity 0.8.21;
+
+/**
+ * @title IRegistry
+ * @dev Interface for Registry
+ */
+interface ICatalog {
```

Comment:
We should break up functionality into multiple files based on functionality so any contract using an interface doesn't have to import the entire function list. Single responsibility makes things easier to reuse too. e.g. `ITrackRegistration`, `IReleaseRegistration`, `ICatalog` (initialization function, etc), and more if you see any.

Reply:
Done.